fronts-client/src/reducers/rootReducer.ts

```diff
@@ -51,7 +52,7 @@ export interface State {
   editor: editorState;
   staleFronts: staleFrontsState;
   form: FormStateMap;
-  optionsModal: OptionsModalState;
+  optionsModal: OptionsModalState | null;
```

Comment:
This was needed for the types to compile. Makes sense as this is not present in the state when there is no modal open

Reply:
Seems sensible!